content/docs/tutorials/at-dude/6-send_dude-screen-app-bar/index.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: codelab
+
+title: "Send Dude Screen AppBar" # Step Name
+description: Creating the UI of the send dude screen # SEO Description for this step Documentation
+
+draft: true # TODO CHANGE THIS TO FALSE WHEN YOU ARE READY TO PUBLISH THE PAGE
```

Comment:
Same thing here, `draft: false`
If you want to merge it as `draft: true`, then just ignore this review. =)

Reply:
@CurtlyCritchlow